app/server.ts

```diff
@@ -0,0 +1,41 @@
+import * as Express from 'express';
+import * as BodyParser from 'body-parser';
+import * as Morgan from 'morgan';
+
+import { PORT } from './modules/util/config';
+import { BasePath as ProjectBasePath, projectRouter } from './modules/project';
+
+export class Server {
+    server: Express.Application;
+    port: string;
+
+    constructor() {
+        this.server = Express();
+        this.port = PORT;
```

Comment:
Ideally you should take port as a parameter in your constructor and keep the default value to something ( may be from config file ) so that this server can be used in other places too ( in case you want to spin up a server for your tests on a random port ? )

- [ ] Expect port as a parameter